organization/voting.md

```diff
@@ -19,14 +26,47 @@ check conway's law: "Any organization that designs a system will produce a desig
 
 5 seems to not introduce too many established structures
 
+```{note}
+I follow the rationale. Now, should the collective start at 5? If not, what is the timeline to gradually get to this figure? Are processes defined elsewhere robust to size fluctuations?
+```
+
 ## Voting processes
 
 collective is composed of 5 people, each 1 vote. votes are cryptographically signed.<br>
 
-decision is taken with majority, and everyone needs to vote. Exceptionally, a decision can be taken by only 4 members if they all agree, i.e. all 4 vote for the same resolution.<br>
+decision is taken with majority, and everyone needs to vote. 
+
+```{note}
+[brainstorming]
+Given assumption above, how are negative votes handled? If vote is sync, we expect all members to be present at a given time and the vote decision YES/NO is taken. If vote is async, we expect all members to have produced a vote by a deadline.
+```
+
+Exceptionally, a decision can be taken by only 4 members if they all agree, i.e. all 4 vote for the same resolution.<br>
 
 collective changes can happen and need to be voted upon
 
 eviction of one person cannot happen with simple majority, it needs to have 4 votes.
 
 collective has multisig authority over the main account (which main account?) (3-of-5, 4-of-5?)
+
+```{note}
+[Brainstorming]
+I would imagine several types of access:
+
+Access to repo for statutes - SOP repository (access governed by voting majority rules). This would be comparable to a set of laws - legislative output.
+
+On top of this, powers and resources to execute the policies. Would this be the main account ? A possible list of critical items:
+* Access to repo for infra
+* Access to secret keys for 
+    * infra & docs & code
+    * third party services (e.g. gandi, bitwarden)
+    * collective crypto assets
+
+=> This account would be accessed to implement the decisions from statutes.
+Access to this would require more `privileged` access than statutes repo - or stronger majority (rewriting SOPs is only thing, accessing all resources another).
+Then, one argue access to this account would need to be done by 'executive branch', not necesarilly the same 'legislative body' who vote on statutes.
+
+Is separation of power needed for an org of 5?. Are the same 5 required to multi-sig (unanimity) after statutes/SOP decisions to access critical resources?
+
+I am getting confused. There is maybe existing litterature (governance or co-op)about these topics.
+``` 
```

Comment:
agreed on all you said. I don't think I have something concrete and specified in mind, these were just ideas thrown at the wall to see what sticks.

As I'd like the growth to be organic, the writing of the processes and accesses to resources should grow and be defined as time passes by and the needs arise.

So our current status is:
- this document reflects a long-term ideal way of organizing the collective but as of now is just a wish, or a guideline
- our current statutes and SOPs are empty.

Over time, we want to transfer the ideas in the manifesto (nice philosophical ideas) to statutes and SOPs (concrete actionable items that can be followed in the day-to-day work organization).
So (being a bit meta here), this PR and the comments we are having on it might lead us to decide that the way we comment on proposed changes is going to be on Github PRs, or it might be something else that seems more suited to how we want to work. When we decide how we want to do it, we write an SOP (or edit a currently existing one), approve it, and then it becomes a "rule".
When we create a new resource (new server in the cloud, email for team members, etc.) that needs to have shared access, we decide at the time that this resource comes into play who has access and how, and describe it in an infrastructure document.